packages/admin/src/table/Table.tsx

```diff
@@ -16,7 +16,7 @@ import { safeColumnGet } from "./safeColumnGet";
 import * as sc from "./Table.sc";
 import { ISortApi, SortDirection } from "./useTableQuerySort";
 
-export function TableBodyRow({ innerRef, ...props }: sc.ITableBodyRowProps) {
```

Comment:
innerRef is deprecated anyway and should not be used anymore 
https://styled-components.com/docs/api#deprecated-innerref-prop